### PR TITLE
daily log end-point

### PR DESCRIPTION
**Pull request by dave-vazquez**
# Description

Creates endpoints to retrieve information from the user's daily food log.

- [x] New feature (non-breaking change which adds functionality)

## Change Status

- [x] Complete, tested, ready to review and merge

# Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

File: routes/dailyLog/dailyLog.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+const {
+  macroRatiosToGrams,
+  applyTimeZones,
+  calculateConsumption,
+} = require("./helper/helper.js");
+
+/********************************************************
+*                   GET USER/DAILY-LOG                  *
+********************************************************/
+router.get("/:user_id/:date/:tz_name_current", mapFirebaseIDtoUserID, async (req, res) => {
```
Comment (israelgonzalezb) [inline routes/dailyLog/dailyLog.js]: Is this route visible to the user or is this just an endpoint we hit?

File: routes/dailyLog/helper/helper.js [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+        log.timeZoneName
+      );
+    }
+  });
+
+  // localizes a UTC-time to a provided time-zone
+  function applyTimeZoneOffset(timeUTC, timeZoneName) {
+    let timeConsumedUTC = moment.utc(timeUTC);
+    return moment.tz(timeConsumedUTC, timeZoneName).format();
+  }
```
Comment (israelgonzalezb) [inline routes/dailyLog/helper/helper.js]: is nice!

File: routes/dailyLog/dailyLog.js (hunk 1 above)
Comment (dave-vazquez) [reply routes/dailyLog/dailyLog.js]: just an endpoint we hit

File: routes/dailyLog/helper/helper.js (hunk 2 above)
Comment (dave-vazquez) [reply routes/dailyLog/helper/helper.js]: 🙇

**Pull request merged**